.circleci/config.yml

```diff
@@ -11,9 +11,15 @@ workflows:
       - architect/push-to-app-catalog:
           context: architect
           name: push-to-app-catalog
-          app_catalog: "control-plane-test-catalog"
+          app_catalog: "control-plane-catalog"
           app_catalog_test: "control-plane-test-catalog"
           chart: "cluster-api-provider-azure"
+          filters:
+            # Only do this when a new tag is created.
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^v.*/
```

Comment:
The difference with now is two fold:
- push to catalog only when there is a new tag, instead of every commit
- actually push to catalog when there is a new tag